google/cloud/security/inventory/pipelines/load_group_members_pipeline.py

```diff
@@ -102,16 +106,29 @@ def _retrieve(self):
                 raise inventory_errors.LoadDataPipelineError(e)
 
             group_members_map.append((group_id, group_members))
+            LOGGER.debug('Retrieved members from {0}: {1}'.format(
+                         group_id,
+                         len(group_members)))
 
         return group_members_map
 
     def run(self):
         """Runs the load GSuite account groups pipeline."""
-        groups_members_map = self._retrieve()
-
-        if isinstance(groups_members_map, list):
-            loadable_group_members = self._transform(groups_members_map)
-            self._load(self.RESOURCE_NAME, loadable_group_members)
-            self._get_loaded_count()
-        else:
-            LOGGER.warn('No group members retrieved.')
+
+        group_ids = self._fetch_groups_from_dao()
+
+        def chunker(seq, size):
```

Comment:
Can we add a simple unit test for this chunker()?  It should be pretty straightforward to feed in a fake seq, and fake size, and then assert on the result.